sideboard/lib/sa/__init__.py

```diff
@@ -143,7 +143,17 @@ def process_result_value(self, value, engine):
     __all__.append('UTCDateTime')
 
 
-def declarative_base(klass):
+# SQLAlchemy doesn't expose its default constructor as a nicely importable
+# function, so we grab it from the function defaults.
+if six.PY2:
+    _spec_args, _spec_varargs, _spec_kwargs, _spec_defaults = inspect.getargspec(declarative.declarative_base)
+else:
+    _declarative_spec = inspect.getfullargspec(declarative.declarative_base)
+    _spec_args, _spec_defaults = _declarative_spec.args, _declarative_spec.defaults
+declarative_base_constructor = dict(zip(reversed(_spec_args), reversed(_spec_defaults)))['constructor']
```

Comment:
I moved this out of the function. Now you can just import `declarative_base_constructor` if you need it elsewhere (Uber has an exact copy of this code block).